docs/REST API/rest-api.md

```diff
@@ -8485,6 +8485,9 @@ Deletes the session specified by ID. When the user associated with the session n
 - [Add package](#add-package)
 - [List App Store apps](#list-app-store-apps)
 - [Add App Store app](#add-app-store-app)
+- [List Fleet maintained apps](#list-fleet-maintained-apps)
```

Comment:
```suggestion
- [List Fleet-maintained apps](#list-fleet-maintained-apps)
```

@mike-j-thomas do we want a hyphen here? And other places we say "Fleet maintained app"

Reply:
@noahtalerman Yes, compound adjectives should be hyphenated `"Fleet-maintained apps" = ✅.` Compound adjectives that contain comparative or superlative modifiers should not be, e.g., `"Most-brilliant software" = ❌.` Likewise, for adverbs that end with -ly, e.g., `"unusually-talented engineer" = ❌`). 🤓